jest.config.mjs

```diff
@@ -6,14 +6,14 @@ const RUN_STANDALONE_TESTS = Boolean(process.env.RUN_STANDALONE_TESTS);
 export default {
   // collectCoverage: ENABLE_COVERAGE,
   collectCoverageFrom: [
-    "<rootDir>/src/**/*.js",
+    "<rootDir>/src/**/*.mjs",
     "!<rootDir>/node_modules/",
     "!<rootDir>/tests_config/",
   ],
   runner: "jest-light-runner",
   transform: {},
-  setupFiles: ["<rootDir>/tests_config/run_spec.js"],
-  testRegex: "jsfmt\\.spec\\.js$|__tests__/.*\\.js$",
+  setupFiles: ["<rootDir>/tests_config/run_spec.mjs"],
+  testRegex: "jsfmt\\.spec\\.mjs$|__tests__/.*\\.mjs$",
```

Comment:
If contributors/PRs accidentally name test files as `.js`, tests will still pass since they won't be tested at all.

Do you think it's a good idea to use `m?js`?

Reply:
>use m?js?
 
Good idea! I'd like that too.